ovos_config/mycroft.conf

```diff
@@ -607,10 +607,20 @@
     "module": "ovos-tts-plugin-server",
     "fallback_module": "ovos-tts-plugin-mimic3-server",
     "ovos-tts-plugin-mimic": {
-        "voice": "ap"
+        "voice": "ap",
+        // persistent cache
+        "persist_cache": false,
```

Comment:
docstr saying this will add every new synth to the persistent cache (note that this cache isnt cleared)